cli/runner.go

```diff
@@ -0,0 +1,296 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+)
+
+const ContainerNetName = "dyo-cli"
```

Comment:
The thing that is running is not the cli (duh), it would look better using the product's name or similar: `dyrectorio-stack`, `dyrectorio-docker` or dev suffix.

Reply:
changed to dyrectorio-stack